getting-started/logiq-paas-deployment/README.md

```diff
@@ -80,7 +80,7 @@ To give you a head start with configuring your Apica Ascent deployment, we've pr
 You can pass the `values.yaml` file with the `helm install` command using the `-f` flag, as shown in the following example.
 
 ```
-helm install logiq --namespace logiq --set global.persistence.storageClass=<storage_class_name> logiq-repo/logiq -f values.small.yaml
+helm install apica-ascent --namespace apica-ascent --set global.persistence.storageClass=<storage_class_name> logiq-repo/logiq -f values.small.yaml
```

Comment:
logiq-repo/logiq should be logiq-repo/apica-ascent

we should change the repo add step to reflect that as well

Reply:
Done, along with some other updates.